src/app/jscaip/Vector.ts

```diff
@@ -27,4 +27,13 @@ export class Vector implements ComparableObject {
         return new Vector(this.x / greatestCommonDivider,
                           this.y / greatestCommonDivider);
     }
+    public combine(otherVector: Vector, times: number = 1): Vector {
+        // return the combined vectors
```

Comment:
mettre le commentaire avant la déclaration, et le formuler plus clairement, par exemple "Adds otherVector to this vector and returns the result"

Reply:
du coup peut-être enlever ce commentaire devenu inutile ?